src/Adapter/ContainerBuilder.php

```diff
@@ -90,6 +91,11 @@ class ContainerBuilder
      */
     public static function getContainer($containerName, $isDebug)
     {
+        if ($containerName === 'admin') {
+            throw new ServiceContainerException(
```

Comment:
It should be considered as a BC since the method now throw an exception

Reply:
@PierreRambaud What do you think it's best then, remove the exception or set it as a BC Break?

Reply:
Set it as BC break :sweat_smile:

Reply:
@PierreRambaud I updated the PR description ;)